[libxcrypt] Build error on x64-linux

Package: libxcrypt:x64-linux@4.4.36#1

**Host Environment**

- Host: x64-linux
- Compiler: GNU 4.8.5
-    vcpkg-tool version: 2024-12-09-1005b78fa1bf1dde1a20c2734cba4ea61ca94d9a
    vcpkg-scripts version: 80d54ff62 2024-12-23 (5 days ago)

**To Reproduce**

`vcpkg install libxcrypt`

**Failure logs**

```
libxcrypt currently requires the following libraries from the system package manager:
    autoconf automake libtool pkg-config
These can be installed on Debian systems via sudo apt install autoconf automake libtool pkg-config
-- Using cached besser82-libxcrypt-v4.4.36.tar.gz.
-- Cleaning sources at /mnt/petrelfs/fangkechen/evaluation/vcpkg/buildtrees/libxcrypt/src/v4.4.36-645fb55159.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /mnt/petrelfs/fangkechen/evaluation/vcpkg/downloads/besser82-libxcrypt-v4.4.36.tar.gz
-- Using source at /mnt/petrelfs/fangkechen/evaluation/vcpkg/buildtrees/libxcrypt/src/v4.4.36-645fb55159.clean
-- Found external ninja('1.11.1.git.kitware.jobserver-1').
-- Getting CMake variables for x64-linux-dbg
-- Getting CMake variables for x64-linux-rel
-- Generating configure for x64-linux
-- Finished generating configure for x64-linux
-- Configuring x64-linux-dbg
-- Configuring x64-linux-rel
-- Building x64-linux-dbg
-- Installing x64-linux-dbg
-- Building x64-linux-rel
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /usr/bin/make V=1 -j 257 -f Makefile all
    Working Directory: /mnt/petrelfs/fangkechen/evaluation/vcpkg/buildtrees/libxcrypt/x64-linux-rel/
    See logs for more information:
      /mnt/petrelfs/fangkechen/evaluation/vcpkg/buildtrees/libxcrypt/build-x64-linux-rel-out.log
      /mnt/petrelfs/fangkechen/evaluation/vcpkg/buildtrees/libxcrypt/build-x64-linux-rel-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_make.cmake:136 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_make.cmake:2 (vcpkg_build_make)
  ports/libxcrypt/portfile.cmake:30 (vcpkg_install_make)
  scripts/ports.cmake:196 (include)



```

<details><summary>/mnt/petrelfs/fangkechen/evaluation/vcpkg/buildtrees/libxcrypt/build-x64-linux-rel-err.log</summary>

```
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c: In function ‘_crypt_gensalt_bcrypt_rn’:
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c:436:10: error: assuming pointer wraparound does not occur when comparing P +- C1 with P +- C2 [-Werror=strict-overflow]
       if (sptr >= end)
          ^
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c:446:10: error: assuming pointer wraparound does not occur when comparing P +- C1 with P +- C2 [-Werror=strict-overflow]
       if (sptr >= end)
          ^
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c:457:3: error: assuming pointer wraparound does not occur when comparing P +- C1 with P +- C2 [-Werror=strict-overflow]
   while (sptr < end);
   ^
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c:436:10: error: assuming pointer wraparound does not occur when comparing P +- C1 with P +- C2 [-Werror=strict-overflow]
       if (sptr >= end)
          ^
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c:446:10: error: assuming pointer wraparound does not occur when comparing P +- C1 with P +- C2 [-Werror=strict-overflow]
       if (sptr >= end)
          ^
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c:457:3: error: assuming pointer wraparound does not occur when comparing P +- C1 with P +- C2 [-Werror=strict-overflow]
   while (sptr < end);
   ^
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c: In function ‘_crypt_gensalt_bcrypt_a_rn’:
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c:436:10: error: assuming pointer wraparound does not occur when comparing P +- C1 with P +- C2 [-Werror=strict-overflow]
       if (sptr >= end)
          ^
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c:446:10: error: assuming pointer wraparound does not occur when comparing P +- C1 with P +- C2 [-Werror=strict-overflow]
       if (sptr >= end)
          ^
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c:457:3: error: assuming pointer wraparound does not occur when comparing P +- C1 with P +- C2 [-Werror=strict-overflow]
   while (sptr < end);
   ^
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c:436:10: error: assuming pointer wraparound does not occur when comparing P +- C1 with P +- C2 [-Werror=strict-overflow]
       if (sptr >= end)
          ^
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c:446:10: error: assuming pointer wraparound does not occur when comparing P +- C1 with P +- C2 [-Werror=strict-overflow]
       if (sptr >= end)
          ^
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c:457:3: error: assuming pointer wraparound does not occur when comparing P +- C1 with P +- C2 [-Werror=strict-overflow]
   while (sptr < end);
   ^
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c: In function ‘_crypt_gensalt_bcrypt_y_rn’:
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c:436:10: error: assuming pointer wraparound does not occur when comparing P +- C1 with P +- C2 [-Werror=strict-overflow]
       if (sptr >= end)
          ^
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c:446:10: error: assuming pointer wraparound does not occur when comparing P +- C1 with P +- C2 [-Werror=strict-overflow]
       if (sptr >= end)
          ^
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c:457:3: error: assuming pointer wraparound does not occur when comparing P +- C1 with P +- C2 [-Werror=strict-overflow]
   while (sptr < end);
   ^
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c:436:10: error: assuming pointer wraparound does not occur when comparing P +- C1 with P +- C2 [-Werror=strict-overflow]
       if (sptr >= end)
          ^
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c:446:10: error: assuming pointer wraparound does not occur when comparing P +- C1 with P +- C2 [-Werror=strict-overflow]
       if (sptr >= end)
          ^
./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c:457:3: error: assuming pointer wraparound does not occur when comparing P +- C1 with P +- C2 [-Werror=strict-overflow]
   while (sptr < end);
   ^
cc1: all warnings being treated as errors
make[1]: *** [lib/libcrypt_la-crypt-bcrypt.lo] Error 1
make: *** [all] Error 2
```
</details>

<details><summary>/mnt/petrelfs/fangkechen/evaluation/vcpkg/buildtrees/libxcrypt/build-x64-linux-rel-out.log</summary>

```
/usr/bin/make  all-am
make[1]: Entering directory `/mnt/petrelfs/fangkechen/evaluation/vcpkg/buildtrees/libxcrypt/x64-linux-rel'
LC_ALL=C /mnt/petrelfs/fangkechen/anaconda3/envs/cognvs/bin/perl ./../src/v4.4.36-645fb55159.clean/build-aux/scripts/gen-libcrypt-map \
  SYMVER_MIN=XCRYPT_2.0 \
  SYMVER_FLOOR=XCRYPT_2.0 \
  COMPAT_ABI=yes \
  ./../src/v4.4.36-645fb55159.clean/lib/libcrypt.map.in > libcrypt.map.T
depbase=`echo lib/gen-des-tables.o | sed 's|[^/]*$|.deps/&|;s|\.o$||'`;\
/usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/gen-des-tables.o -MD -MP -MF $depbase.Tpo -c -o lib/gen-des-tables.o ./../src/v4.4.36-645fb55159.clean/lib/gen-des-tables.c &&\
mv -f $depbase.Tpo $depbase.Po
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-alg-des-tables.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-des-tables.Tpo -c -o lib/libcrypt_la-alg-des-tables.lo `test -f 'lib/alg-des-tables.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/alg-des-tables.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-alg-des.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-des.Tpo -c -o lib/libcrypt_la-alg-des.lo `test -f 'lib/alg-des.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/alg-des.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-alg-gost3411-2012-core.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-gost3411-2012-core.Tpo -c -o lib/libcrypt_la-alg-gost3411-2012-core.lo `test -f 'lib/alg-gost3411-2012-core.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/alg-gost3411-2012-core.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-alg-gost3411-2012-hmac.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-gost3411-2012-hmac.Tpo -c -o lib/libcrypt_la-alg-gost3411-2012-hmac.lo `test -f 'lib/alg-gost3411-2012-hmac.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/alg-gost3411-2012-hmac.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-alg-hmac-sha1.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-hmac-sha1.Tpo -c -o lib/libcrypt_la-alg-hmac-sha1.lo `test -f 'lib/alg-hmac-sha1.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/alg-hmac-sha1.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-alg-md4.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-md4.Tpo -c -o lib/libcrypt_la-alg-md4.lo `test -f 'lib/alg-md4.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/alg-md4.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-alg-md5.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-md5.Tpo -c -o lib/libcrypt_la-alg-md5.lo `test -f 'lib/alg-md5.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/alg-md5.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-alg-sha1.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-sha1.Tpo -c -o lib/libcrypt_la-alg-sha1.lo `test -f 'lib/alg-sha1.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/alg-sha1.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-alg-sha256.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-sha256.Tpo -c -o lib/libcrypt_la-alg-sha256.lo `test -f 'lib/alg-sha256.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/alg-sha256.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-alg-sha512.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-sha512.Tpo -c -o lib/libcrypt_la-alg-sha512.lo `test -f 'lib/alg-sha512.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/alg-sha512.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-alg-yescrypt-common.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-yescrypt-common.Tpo -c -o lib/libcrypt_la-alg-yescrypt-common.lo `test -f 'lib/alg-yescrypt-common.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/alg-yescrypt-common.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-alg-yescrypt-opt.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-yescrypt-opt.Tpo -c -o lib/libcrypt_la-alg-yescrypt-opt.lo `test -f 'lib/alg-yescrypt-opt.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/alg-yescrypt-opt.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-crypt-bcrypt.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-bcrypt.Tpo -c -o lib/libcrypt_la-crypt-bcrypt.lo `test -f 'lib/crypt-bcrypt.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/crypt-bcrypt.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-crypt-des.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-des.Tpo -c -o lib/libcrypt_la-crypt-des.lo `test -f 'lib/crypt-des.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/crypt-des.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-crypt-gensalt-static.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-gensalt-static.Tpo -c -o lib/libcrypt_la-crypt-gensalt-static.lo `test -f 'lib/crypt-gensalt-static.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/crypt-gensalt-static.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-crypt-gost-yescrypt.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-gost-yescrypt.Tpo -c -o lib/libcrypt_la-crypt-gost-yescrypt.lo `test -f 'lib/crypt-gost-yescrypt.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/crypt-gost-yescrypt.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-crypt-md5.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-md5.Tpo -c -o lib/libcrypt_la-crypt-md5.lo `test -f 'lib/crypt-md5.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/crypt-md5.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-crypt-nthash.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-nthash.Tpo -c -o lib/libcrypt_la-crypt-nthash.lo `test -f 'lib/crypt-nthash.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/crypt-nthash.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-crypt-pbkdf1-sha1.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-pbkdf1-sha1.Tpo -c -o lib/libcrypt_la-crypt-pbkdf1-sha1.lo `test -f 'lib/crypt-pbkdf1-sha1.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/crypt-pbkdf1-sha1.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-crypt-scrypt.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-scrypt.Tpo -c -o lib/libcrypt_la-crypt-scrypt.lo `test -f 'lib/crypt-scrypt.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/crypt-scrypt.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-crypt-sha256.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-sha256.Tpo -c -o lib/libcrypt_la-crypt-sha256.lo `test -f 'lib/crypt-sha256.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/crypt-sha256.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-crypt-sha512.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-sha512.Tpo -c -o lib/libcrypt_la-crypt-sha512.lo `test -f 'lib/crypt-sha512.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/crypt-sha512.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-crypt-static.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-static.Tpo -c -o lib/libcrypt_la-crypt-static.lo `test -f 'lib/crypt-static.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/crypt-static.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-crypt-sunmd5.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-sunmd5.Tpo -c -o lib/libcrypt_la-crypt-sunmd5.lo `test -f 'lib/crypt-sunmd5.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/crypt-sunmd5.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-crypt-yescrypt.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-yescrypt.Tpo -c -o lib/libcrypt_la-crypt-yescrypt.lo `test -f 'lib/crypt-yescrypt.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/crypt-yescrypt.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-crypt.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt.Tpo -c -o lib/libcrypt_la-crypt.lo `test -f 'lib/crypt.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/crypt.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-util-base64.lo -MD -MP -MF lib/.deps/libcrypt_la-util-base64.Tpo -c -o lib/libcrypt_la-util-base64.lo `test -f 'lib/util-base64.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/util-base64.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-util-gensalt-sha.lo -MD -MP -MF lib/.deps/libcrypt_la-util-gensalt-sha.Tpo -c -o lib/libcrypt_la-util-gensalt-sha.lo `test -f 'lib/util-gensalt-sha.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/util-gensalt-sha.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-util-get-random-bytes.lo -MD -MP -MF lib/.deps/libcrypt_la-util-get-random-bytes.Tpo -c -o lib/libcrypt_la-util-get-random-bytes.lo `test -f 'lib/util-get-random-bytes.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/util-get-random-bytes.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-util-make-failure-token.lo -MD -MP -MF lib/.deps/libcrypt_la-util-make-failure-token.Tpo -c -o lib/libcrypt_la-util-make-failure-token.lo `test -f 'lib/util-make-failure-token.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/util-make-failure-token.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-util-xbzero.lo -MD -MP -MF lib/.deps/libcrypt_la-util-xbzero.Tpo -c -o lib/libcrypt_la-util-xbzero.lo `test -f 'lib/util-xbzero.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/util-xbzero.c
/bin/sh ./libtool  --tag=CC   --mode=compile /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean  -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -MT lib/libcrypt_la-util-xstrcpy.lo -MD -MP -MF lib/.deps/libcrypt_la-util-xstrcpy.Tpo -c -o lib/libcrypt_la-util-xstrcpy.lo `test -f 'lib/util-xstrcpy.c' || echo './../src/v4.4.36-645fb55159.clean/'`lib/util-xstrcpy.c
./../src/v4.4.36-645fb55159.clean/build-aux/scripts/move-if-change libcrypt.map.T libcrypt.map
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-crypt-gost-yescrypt.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-gost-yescrypt.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/crypt-gost-yescrypt.c -o lib/libcrypt_la-crypt-gost-yescrypt.o
echo timestamp > libcrypt.map.stamp
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-alg-yescrypt-opt.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-yescrypt-opt.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/alg-yescrypt-opt.c -o lib/libcrypt_la-alg-yescrypt-opt.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-alg-hmac-sha1.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-hmac-sha1.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/alg-hmac-sha1.c -o lib/libcrypt_la-alg-hmac-sha1.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-alg-des-tables.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-des-tables.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/alg-des-tables.c -o lib/libcrypt_la-alg-des-tables.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-alg-md4.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-md4.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/alg-md4.c -o lib/libcrypt_la-alg-md4.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-alg-sha1.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-sha1.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/alg-sha1.c -o lib/libcrypt_la-alg-sha1.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-alg-gost3411-2012-hmac.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-gost3411-2012-hmac.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/alg-gost3411-2012-hmac.c -o lib/libcrypt_la-alg-gost3411-2012-hmac.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-crypt-sha512.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-sha512.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/crypt-sha512.c -o lib/libcrypt_la-crypt-sha512.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-crypt-scrypt.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-scrypt.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/crypt-scrypt.c -o lib/libcrypt_la-crypt-scrypt.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-alg-yescrypt-common.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-yescrypt-common.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/alg-yescrypt-common.c -o lib/libcrypt_la-alg-yescrypt-common.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-alg-sha256.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-sha256.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/alg-sha256.c -o lib/libcrypt_la-alg-sha256.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-alg-gost3411-2012-core.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-gost3411-2012-core.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/alg-gost3411-2012-core.c -o lib/libcrypt_la-alg-gost3411-2012-core.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-crypt-static.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-static.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/crypt-static.c -o lib/libcrypt_la-crypt-static.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-crypt-gensalt-static.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-gensalt-static.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/crypt-gensalt-static.c -o lib/libcrypt_la-crypt-gensalt-static.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-crypt-yescrypt.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-yescrypt.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/crypt-yescrypt.c -o lib/libcrypt_la-crypt-yescrypt.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-crypt-md5.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-md5.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/crypt-md5.c -o lib/libcrypt_la-crypt-md5.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-alg-des.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-des.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/alg-des.c -o lib/libcrypt_la-alg-des.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-alg-md5.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-md5.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/alg-md5.c -o lib/libcrypt_la-alg-md5.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-crypt-sha256.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-sha256.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/crypt-sha256.c -o lib/libcrypt_la-crypt-sha256.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-crypt.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/crypt.c -o lib/libcrypt_la-crypt.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-alg-sha512.lo -MD -MP -MF lib/.deps/libcrypt_la-alg-sha512.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/alg-sha512.c -o lib/libcrypt_la-alg-sha512.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-crypt-des.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-des.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/crypt-des.c -o lib/libcrypt_la-crypt-des.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-crypt-bcrypt.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-bcrypt.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/crypt-bcrypt.c -o lib/libcrypt_la-crypt-bcrypt.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-crypt-nthash.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-nthash.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/crypt-nthash.c -o lib/libcrypt_la-crypt-nthash.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-util-xstrcpy.lo -MD -MP -MF lib/.deps/libcrypt_la-util-xstrcpy.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/util-xstrcpy.c -o lib/libcrypt_la-util-xstrcpy.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-crypt-sunmd5.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-sunmd5.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/crypt-sunmd5.c -o lib/libcrypt_la-crypt-sunmd5.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-crypt-pbkdf1-sha1.lo -MD -MP -MF lib/.deps/libcrypt_la-crypt-pbkdf1-sha1.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/crypt-pbkdf1-sha1.c -o lib/libcrypt_la-crypt-pbkdf1-sha1.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-util-xbzero.lo -MD -MP -MF lib/.deps/libcrypt_la-util-xbzero.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/util-xbzero.c -o lib/libcrypt_la-util-xbzero.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-util-base64.lo -MD -MP -MF lib/.deps/libcrypt_la-util-base64.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/util-base64.c -o lib/libcrypt_la-util-base64.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-util-make-failure-token.lo -MD -MP -MF lib/.deps/libcrypt_la-util-make-failure-token.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/util-make-failure-token.c -o lib/libcrypt_la-util-make-failure-token.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-util-get-random-bytes.lo -MD -MP -MF lib/.deps/libcrypt_la-util-get-random-bytes.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/util-get-random-bytes.c -o lib/libcrypt_la-util-get-random-bytes.o
libtool: compile:  /usr/bin/cc -std=gnu11 -DHAVE_CONFIG_H -I. -I./../src/v4.4.36-645fb55159.clean -I./../src/v4.4.36-645fb55159.clean/lib -DIN_LIBCRYPT -DNDEBUG -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -MT lib/libcrypt_la-util-gensalt-sha.lo -MD -MP -MF lib/.deps/libcrypt_la-util-gensalt-sha.Tpo -c ./../src/v4.4.36-645fb55159.clean/lib/util-gensalt-sha.c -o lib/libcrypt_la-util-gensalt-sha.o
/bin/sh ./libtool  --tag=CC   --mode=link /usr/bin/cc -std=gnu11 -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror  -fPIC -O3 -Wl,-z,relro -Wl,-z,now -L/mnt/petrelfs/fangkechen/evaluation/vcpkg/installed/x64-linux/lib -o lib/gen-des-tables lib/gen-des-tables.o  
mv -f lib/.deps/libcrypt_la-crypt-gensalt-static.Tpo lib/.deps/libcrypt_la-crypt-gensalt-static.Plo
mv -f lib/.deps/libcrypt_la-alg-hmac-sha1.Tpo lib/.deps/libcrypt_la-alg-hmac-sha1.Plo
mv -f lib/.deps/libcrypt_la-util-xbzero.Tpo lib/.deps/libcrypt_la-util-xbzero.Plo
mv -f lib/.deps/libcrypt_la-util-base64.Tpo lib/.deps/libcrypt_la-util-base64.Plo
mv -f lib/.deps/libcrypt_la-crypt-yescrypt.Tpo lib/.deps/libcrypt_la-crypt-yescrypt.Plo
mv -f lib/.deps/libcrypt_la-crypt-scrypt.Tpo lib/.deps/libcrypt_la-crypt-scrypt.Plo
mv -f lib/.deps/libcrypt_la-crypt-nthash.Tpo lib/.deps/libcrypt_la-crypt-nthash.Plo
mv -f lib/.deps/libcrypt_la-crypt-static.Tpo lib/.deps/libcrypt_la-crypt-static.Plo
mv -f lib/.deps/libcrypt_la-util-make-failure-token.Tpo lib/.deps/libcrypt_la-util-make-failure-token.Plo
mv -f lib/.deps/libcrypt_la-util-xstrcpy.Tpo lib/.deps/libcrypt_la-util-xstrcpy.Plo
mv -f lib/.deps/libcrypt_la-crypt-md5.Tpo lib/.deps/libcrypt_la-crypt-md5.Plo
mv -f lib/.deps/libcrypt_la-crypt.Tpo lib/.deps/libcrypt_la-crypt.Plo
mv -f lib/.deps/libcrypt_la-alg-des.Tpo lib/.deps/libcrypt_la-alg-des.Plo
mv -f lib/.deps/libcrypt_la-util-gensalt-sha.Tpo lib/.deps/libcrypt_la-util-gensalt-sha.Plo
mv -f lib/.deps/libcrypt_la-alg-des-tables.Tpo lib/.deps/libcrypt_la-alg-des-tables.Plo
mv -f lib/.deps/libcrypt_la-crypt-gost-yescrypt.Tpo lib/.deps/libcrypt_la-crypt-gost-yescrypt.Plo
libtool: link: /usr/bin/cc -std=gnu11 -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -Werror -fPIC -O3 -Wl,-z -Wl,relro -Wl,-z -Wl,now -o lib/gen-des-tables lib/gen-des-tables.o   -L/mnt/petrelfs/fangkechen/evaluation/vcpkg/installed/x64-linux/lib
mv -f lib/.deps/libcrypt_la-crypt-des.Tpo lib/.deps/libcrypt_la-crypt-des.Plo
mv -f lib/.deps/libcrypt_la-alg-gost3411-2012-hmac.Tpo lib/.deps/libcrypt_la-alg-gost3411-2012-hmac.Plo
mv -f lib/.deps/libcrypt_la-crypt-pbkdf1-sha1.Tpo lib/.deps/libcrypt_la-crypt-pbkdf1-sha1.Plo
mv -f lib/.deps/libcrypt_la-crypt-sha256.Tpo lib/.deps/libcrypt_la-crypt-sha256.Plo
mv -f lib/.deps/libcrypt_la-alg-md5.Tpo lib/.deps/libcrypt_la-alg-md5.Plo
mv -f lib/.deps/libcrypt_la-alg-md4.Tpo lib/.deps/libcrypt_la-alg-md4.Plo
mv -f lib/.deps/libcrypt_la-alg-sha1.Tpo lib/.deps/libcrypt_la-alg-sha1.Plo
mv -f lib/.deps/libcrypt_la-crypt-sha512.Tpo lib/.deps/libcrypt_la-crypt-sha512.Plo
mv -f lib/.deps/libcrypt_la-alg-gost3411-2012-core.Tpo lib/.deps/libcrypt_la-alg-gost3411-2012-core.Plo
mv -f lib/.deps/libcrypt_la-crypt-sunmd5.Tpo lib/.deps/libcrypt_la-crypt-sunmd5.Plo
mv -f lib/.deps/libcrypt_la-util-get-random-bytes.Tpo lib/.deps/libcrypt_la-util-get-random-bytes.Plo
mv -f lib/.deps/libcrypt_la-alg-sha512.Tpo lib/.deps/libcrypt_la-alg-sha512.Plo
mv -f lib/.deps/libcrypt_la-alg-yescrypt-common.Tpo lib/.deps/libcrypt_la-alg-yescrypt-common.Plo
mv -f lib/.deps/libcrypt_la-alg-sha256.Tpo lib/.deps/libcrypt_la-alg-sha256.Plo
mv -f lib/.deps/libcrypt_la-alg-yescrypt-opt.Tpo lib/.deps/libcrypt_la-alg-yescrypt-opt.Plo
make[1]: Leaving directory `/mnt/petrelfs/fangkechen/evaluation/vcpkg/buildtrees/libxcrypt/x64-linux-rel'
```
</details>

--- a/ports/libxcrypt/portfile.cmake
+++ b/ports/libxcrypt/portfile.cmake
@@ -15,11 +15,19 @@ if(NOT LIBTOOL_BIN)
     message(FATAL_ERROR "${PORT} requires libtool from the system package manager (example: \"sudo apt install libtool\")")
 endif()
 
+vcpkg_download_distfile(PATCH_FIX_ERROR_STRICT_OVERFLOW
+    URLS https://github.com/besser82/libxcrypt/commit/7fc153170ea6c2938c0392794778de7ec995f8f9.patch?full_index=1
+    SHA512 55f4709c52f6d0a29f159348821c06e3f5df0fae83f487b7a52ce61cd3f6a3a0f48023159fb7029d0a5a3decee36ac6a429cc91e23a83996a3265d681fa11929
+    FILENAME besser82-libxcrypt-7fc153170ea6c2938c0392794778de7ec995f8f9.patch
+)
+
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO besser82/libxcrypt
     REF "v${VERSION}"
     SHA512 61e5e393654f37775457474d4170098314879ee79963d423c1c461e80dc5dc74f0c161dd8754f016ce96109167be6c580ad23994fa1d2c38c54b96e602f3aece
+    PATCHES
+        "${PATCH_FIX_ERROR_STRICT_OVERFLOW}"
 )
 
 vcpkg_configure_make(

--- a/ports/libxcrypt/vcpkg.json
+++ b/ports/libxcrypt/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "libxcrypt",
   "version": "4.4.36",
-  "port-version": 1,
+  "port-version": 2,
   "description": "libxcrypt is a modern library for one-way hashing of passwords. On Linux-based systems, by default libxcrypt will be binary backward compatible with the libcrypt.so.1 shipped as part of the GNU C Library.",
   "homepage": "https://github.com/besser82/libxcrypt",
   "license": "LGPL-2.1-only",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5382,7 +5382,7 @@
     },
     "libxcrypt": {
       "baseline": "4.4.36",
-      "port-version": 1
+      "port-version": 2
     },
     "libxcvt": {
       "baseline": "0.1.2",

--- a/versions/l-/libxcrypt.json
+++ b/versions/l-/libxcrypt.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "825ff0426ce377f1e386f7effd1f3dfb5e218a64",
+      "version": "4.4.36",
+      "port-version": 2
+    },
+    {
       "git-tree": "89d7de97e87cb0eb10479d47ec43e1cc732b2734",
       "version": "4.4.36",
       "port-version": 1

